doc/howto/images_create.md

```diff
@@ -3,3 +3,102 @@
 
 - Publish from an instance or snapshot
 - Use distrobuilder
+
+
+
+### Publishing an instance or snapshot as a new image
+
+An instance or one of its snapshots can be turned into a new image.
+This is done on the CLI with `lxc publish`.
+
+When doing this, you will most likely first want to cleanup metadata and
+templates on the instance you're publishing using the `lxc config metadata`
+and `lxc config template` commands. You will also want to remove any
+instance-specific state like host SSH keys, `dbus/systemd machine-id`, ...
```

Comment:
I think its important to say "inside the instance" here to clarify these are not LXD settings, e.g.

> You will also want to remove any
instance-specific state inside the instance, such as host SSH keys, `dbus/systemd machine-id`, etc.